components/status-indicator/test/status-indicator.visual-diff.js

```diff
@@ -0,0 +1,42 @@
+const puppeteer = require('puppeteer');
+const VisualDiff = require('@brightspace-ui/visual-diff');
+
+describe('d2l-status-indicator', function() {
+
+	const visualDiff = new VisualDiff('status-indicator', __dirname);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await browser.newPage();
+		await page.setViewport({width: 800, height: 800, deviceScaleFactor: 2});
+		await page.goto(`${visualDiff.getBaseUrl()}/components/status-indicator/test/status-indicator.visual-diff.html`, {waitUntil: ['networkidle0', 'load']});
+		await page.bringToFront();
+	});
+
+	beforeEach(async() => {
+		await visualDiff.resetFocus(page);
```

Comment:
I don't _think_ you should need this `resetFocus`, since focus is not being applied to the element being diff'd. This method is helpful in cases where the test applies focus to the element, but we're not refreshing the page, and we need a convenient way to remove focus from the element for subsequent tests.